# HG changeset patch
# Parent  3c1187a6876d8ff634acc7f75b5d8ae23feda534
Ignore my packet captures directory

I never found myself looking for an older pcap, what we really need is a
bulb simulator and test scenarios.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -1,3 +1,4 @@
 .*\.sw[a-z]$
 .*\.py[co]$
 ^build
+^pcaps
